resource.address/http/src/main/java/org/kaazing/gateway/resource/address/http/HttpResourceAddress.java

```diff
@@ -248,6 +256,12 @@ private HttpKeepAliveTimeoutOption() {
         }
     }
 
+    private static final class HttpKeepAliveMaxConnectionsOption extends HttpResourceOption<Integer> {
+        private HttpKeepAliveMaxConnectionsOption() {
+            super(Kind.KEEP_ALIVE_MAX_CONNECTIONS, "keepalive.max.connections", 5);
```

Comment:
Is it possible to use a constant for the default so that there is only one constant for here, the ConnectOptions, etc.?